packages/button/src/Button/Button.tsx

```diff
@@ -84,7 +84,8 @@ const Root = React.forwardRef<RootRef, RootProps>(
             buttonSizeMap[layout][size],
             className,
           )}
-          data-button-layout={layout}
+          data-tgph-button
+          data-tgph-button-layout={layout}
```

Comment:
Used for granular css targeting. Also added `tgph`prefix to avoid future collisions.